Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -129,6 +134,7 @@ export default class ReportsView extends Vue {
     }
 
     private get isDownloadDisabled(): boolean {
+        this.logger.debug(`Report Component Name: ${this.reportComponentName}`);
```

Comment:
Is this in the right place?

Reply:
Yes since it will get invoked every time the report component name changes (as well as the other ones too). I thought about adding a watch just to log this as well

Reply:
I think a watch might make the intent clearer, but it's not a big deal.

Reply:
Should only be used for debugging